src/commands/birthdays.js

```diff
@@ -1,10 +1,44 @@
-const Util = require('../utils/utils.js')
-const Item = require('../models/Member.js')
-const ChannelRepository = require('../repositories/server-repository')
+const Util = require("../utils/utils.js");
+const ServerRepository = require("../repositories/server-repository");
+const moment = require("moment");
 
 module.exports = {
-    name: 'birthdays',
-    description: 'Lists all users birthdays',
-    // execute: async (message, args) => {
-    // },
-}
+  name: "birthdays",
+  description: "Lists all users birthdays",
+  execute: async (message, args) => {
+    let channel = message.channel;
+
+    if (args.length !== 0) {
```

Comment:
Why is there a validation for arguments? birthdays should not have arguments

Reply:
Oh my god! I'm so sorry, I read it completely backwards. That's fine, my bad

Reply:
No worries! It was still a very legit concern, I just assumed you wanted the command to fail in case of parameters but in the end it's a choice :)